otter/cloud_client.py

```diff
@@ -1051,3 +1051,62 @@ def publish_to_cloudfeeds(event, log=None):
             'content-type': ['application/vnd.rackspace.atom+json']},
         data=event, log=log, success_pred=has_code(201),
         json_response=False)
+
+
+# ----- Cloud orchestration requests -----
+def list_stacks_all(parameters=None):
+    """
+    List Heat stacks.
+
+    :ivar dict parameters: Query parameters to include.
+
+    Succeed on 200.
+
+    :return: a `tuple` of (:obj:`twisted.web.client.Response`, JSON `dict`)
+    """
+
+    # TODO Actually implement batch handling
+    return (
+        service_request(
+            ServiceType.CLOUD_ORCHESTRATION,
+            'GET', 'stacks',
+            params=parameters)
+        .on(log_success_response('request-list-stacks-all', identity))
+        .on(lambda (response, body): body['stacks'])
+    )
+
+
+def create_stack(stack_args):
+    """
+    Create a stack using Heat.
+
+    :ivar dict stack_args:  The dictionary to pass to Heat specifying how
+        the stack should be built.
+
+    :return: a `tuple` of (:obj:`twisted.web.client.Response`, JSON `dict`)
```

Comment:
I recommend just making this return the body too, no need to return the Response object. i.e., just return the second element of this tuple, like you did in `list_stacks_all`.